docker-compose.yml

```diff
@@ -20,6 +20,8 @@ services:
       - .:/app
       - results:/results
     entrypoint: ["celery", "--app", "quarry.web.worker", "worker", ]
+    extra_hosts:
+    - "host.docker.internal:host-gateway"
```

Comment:
Do we need to add this for the web container as well?

Reply:
I don't think so. Replica connections/queries happen through the worker.